### ISSUE TITLE
Exception on broker shutdown

### ISSUE THREAD

**Issue by jangalinski** [zeebe-io/spring-zeebe#30]
running the broker via broker.gsh, on shutdown (strg-c), this is logged:

```
2017-12-07 14:16:12.723  INFO 44558 --- [runner-broker-2] io.zeebe.logstreams                      : Creation of snapshot deployment took 5 ms.
Exception in thread "actor-scheduler-broker" java.lang.NullPointerException
        at java.util.HashMap.putMapEntries(HashMap.java:500)
        at java.util.HashMap.putAll(HashMap.java:784)
        at java.util.Collections$SynchronizedMap.putAll(Collections.java:2594)
        at ch.qos.logback.classic.util.LogbackMDCAdapter.setContextMap(LogbackMDCAdapter.java:197)
        at org.slf4j.MDC.setContextMap(MDC.java:264)
        at io.zeebe.util.actor.ActorSchedulerRunnable.run(ActorSchedulerRunnable.java:118)
        at java.lang.Thread.run(Thread.java:748)
Exception in thread "actor-runner-broker-2" java.lang.NullPointerException
        at java.util.HashMap.putMapEntries(HashMap.java:500)
        at java.util.HashMap.putAll(HashMap.java:784)
        at java.util.Collections$SynchronizedMap.putAll(Collections.java:2594)
        at ch.qos.logback.classic.util.LogbackMDCAdapter.setContextMap(LogbackMDCAdapter.java:197)
        at org.slf4j.MDC.setContextMap(MDC.java:264)
        at io.zeebe.util.actor.ActorRunner.run(ActorRunner.java:85)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Exception in thread "actor-runner-broker-1" java.lang.NullPointerException
        at java.util.HashMap.putMapEntries(HashMap.java:500)
        at java.util.HashMap.putAll(HashMap.java:784)
        at java.util.Collections$SynchronizedMap.putAll(Collections.java:2594)
        at ch.qos.logback.classic.util.LogbackMDCAdapter.setContextMap(LogbackMDCAdapter.java:197)
        at org.slf4j.MDC.setContextMap(MDC.java:264)
        at io.zeebe.util.actor.ActorRunner.run(ActorRunner.java:85)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Exception in thread "actor-runner-service-container" java.lang.NullPointerException
        at java.util.HashMap.putMapEntries(HashMap.java:500)
        at java.util.HashMap.putAll(HashMap.java:784)
        at java.util.Collections$SynchronizedMap.putAll(Collections.java:2594)
        at ch.qos.logback.classic.util.LogbackMDCAdapter.setContextMap(LogbackMDCAdapter.java:197)
        at org.slf4j.MDC.setContextMap(MDC.java:264)
        at io.zeebe.util.actor.ActorRunner.run(ActorRunner.java:85)
        at java.lang.Thread.run(Thread.java:748)
2017-12-07 14:16:12.821  INFO 44558 --- [       Thread-2] io.zeebe.broker.system                   : Broker closed

```

**Comment by menski** [comment]
This should be fixed with https://github.com/zeebe-io/zeebe/issues/606 which is part of 0.7.0

**Comment by jangalinski** [comment]
fixed with 0.7.0
